client/src/app/review-medication/review-medication.component.ts

```diff
@@ -8,7 +8,6 @@ import { MedicationComment } from '../MedicationComment';
 import { MedicationDescription } from '../MedicationDescription';
 import { Sanitizer } from '@angular/core';
 import { SecurityContext } from '@angular/core';
-import { LoginComponent } from '../login/login.component';
```

Comment:
Is this related to the widget template task or is this just tidying up?

Reply:
I think that was just a change i had that slipped into the commit. Just tidy up of the component

Reply:
No worries - will disregard for now ;)